utilities/xmlrpcpp/include/xmlrpcpp/XmlRpcServer.h

```diff
@@ -125,10 +125,6 @@ namespace XmlRpc {
 
     // Minimum number of free file descriptors before rejecting clients.
     static const int FREE_FD_BUFFER;
-#ifndef _WINDOWS
```

Comment:
Oh, and you definitely can't remove anything from header files. That would change API and ABI which is forbidden for released ROS versions.

Reply:
Fixed

Reply:
Hmm, I think you should drop the local `poll_fds` variable and use the one from the header instead. It is a protected variable, so there might be downstream code expecting to find some FDs in it. Would that make sens?

Reply:
Oh you are right, it is a protected member. Will fix it.

Reply:
fixed

Reply:
now it should be correct

Reply:
But actually right now it is not used to hold available file descriptors as this can also change over time

Reply:
Although this member is protected and could in theory be used in successor classes it would not make any sense. These FDs would not be usable in any way. There is no guarantee that they are still available later. Their only purpose was to count the number of available FDs at the time of creation.
This code piece was very questionable from the beginning. The last change to it was supposed to do exactly what it does now with this PR but actually missed the point.

Reply:
You're right, the possibility of using this member is so low that I don't think it is worth it doing anything else...